fluentd/lib/in_tail_metric/write_watcher.rb

```diff
@@ -0,0 +1,72 @@
+#
```

Comment:
WriteWatcher is no longer needed, delete this file and remove all references to it.